src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Utilities/BidirectionalMap.cs

```diff
@@ -115,5 +115,31 @@ public int Count
 
             return default;
         }
+
+        public TValue this[TKey key]
```

Comment:
these intentionally did nto exist, because they are ambiguous if you have `BidiMap<X, X>`.  client needs to specify if they're looking up by key or by value :)

Reply:
IMO, that would still be better than having to use GetValueOrDefault and handling the default return case.